pkg/component/nodemanagement/machinecontrollermanager/provider.go

```diff
@@ -12,6 +12,8 @@ import (
 	vpaautoscalingv1 "k8s.io/autoscaler/vertical-pod-autoscaler/pkg/apis/autoscaling.k8s.io/v1"
 	"k8s.io/utils/ptr"
 
+	extensionscontroller "github.com/gardener/gardener/extensions/pkg/controller"
```

Comment:
I don't think this kind of import is nice. We don't want to import `extensions` from `pkg` unless really needed.